.github/workflows/windows.yml

```diff
@@ -123,3 +123,58 @@ jobs:
       - name: Test
         run: |
           python dev.py test -j2 --mode full
+
+
+  #############################################################################
+  full_build_sdist_wheel:
+    # Python 3.11
+    # python -m build && pip install dist/scipy*.whl
+    # 'full' test run,
+    # -Duse-pythran=false
+    # TODO: enable ILP64 once possible
+    name: cp311 (build sdist + wheel), full, no pythran
+    if: "github.repository == 'scipy/scipy' || github.repository == ''"
+    runs-on: windows-2019
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
+          cache: 'pip'
+          cache-dependency-path: 'environment.yml'
+
+      - name: Win_amd64 - install rtools
+        run: |
+          # mingw-w64
+          choco install rtools --no-progress
+          echo "c:\rtools40\ucrt64\bin;" >> $env:GITHUB_PATH
+
+      - name: Install OpenBLAS
+        shell: bash
+        run: |
+          set -xe
+          bash tools/wheels/cibw_before_build_win.sh .
+          echo "PKG_CONFIG_PATH=c:\opt\64\lib\pkgconfig;" >> $GITHUB_ENV
+
+      - name: pip-packages
+        run: |
+          python -m pip install build delvewheel numpy cython pybind11 meson-python meson ninja pytest pytest-xdist pytest-timeout pooch rich_click click doit pydevtool
```

Comment:
one more comment: `dev.py` isn't used in this job, so you can remove the last four packages from this line